apollo-router/src/axum_factory/axum_http_server_factory.rs

```diff
@@ -97,6 +97,7 @@ where
                     let health = Health {
                         status: HealthStatus::Up,
                     };
+                    tracing::trace!(?health, "health check");
```

Comment:
Don't you think it could be useful to also log the raw request in attributes ?

Reply:
done: b4b5b7454c91f42176ae9bdf1ce1d91b671dc623